src/main/webapp/src/app/create-gardens-form/create-gardens.component.html

```diff
@@ -0,0 +1,35 @@
+<mat-card class="mat-elevation-z8 form" >
+	<mat-card-title>
+		Create Garden Form
+	</mat-card-title>
+
+	<mat-card-content>
+
+		<mat-form-field>
+			<input matInput placeholder="Name of Garden">
+		</mat-form-field><br/>
+
+		<mat-form-field>
+			<input matInput placeholder="Name of Admin">
+		</mat-form-field><br/>
+
+    <mat-form-field>
+			<input matInput placeholder="Address of Garden">
+		</mat-form-field><br/>
+
+    <mat-form-field>
+      <textarea matInput placeholder="Garden Bio"></textarea>
+		</mat-form-field><br/>
+
+    <mat-form-field>
+      <textarea matInput placeholder="Plants Currently In Garden"></textarea>
+		</mat-form-field><br/>
+
+    
+ <button mat-button class="button">Create Garden</button>
+	
+
+	</mat-card-content>
```

Comment:
Are you running the linter and formatter here? Because the linter should automatically take care of this?

Reply:
Done